File: Modul-2/Soal-3/PRAK203-2410817220012-NOOR-KHALISA.c
#include <stdio.h>

int main(){
    float a, b, i, j, x, y;

    printf("Masukkan nilai a, b, i, j, x, dan y secara berurutan (pisahkan dengan spasi): ");
    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);

    float hasil = (((a - b) * i) / j) - (x + y);
    printf("Hasil: %.3f", hasil);

    return 0;
}

File: Modul-2/Soal-3/PRAK203-2410817220012-NOOR-KHALISA.py
def hitung(a, b, i, j, x, y):
    hasil = (((a - b) * i) / j) - (x + y)
    return hasil
input = input("Masukkan nilai a, b, i, j, x, dan y secara berurutan (pisahkan dengan spasi): ")
a, b, i, j, x, y = map(float, input.split())

hasil = hitung(a, b, i, j, x, y)

print(f"Hasil: {hasil:.3f}")